bazel/README.md

```diff
@@ -708,6 +708,8 @@ The extensions enabled by default can be disabled by adding the following parame
 `envoy.wasm.runtime.v8` extension, add `--//source/extensions/wasm_runtime/v8:enabled=false`.
 Note not all extensions can be disabled.
 
+To enable a specific wasm runtime, you'll need to pass `--define wasm=[wasm_runtime]`, e.g. `--define wasm=wasmtime` to enable the [wasmtime](https://wasmtime.dev/) runtime. The list of runtimes available can be found in the [bazel/BUILD](https://github.com/envoyproxy/envoy/blob/0a587f23b21ab34b28d9c53428af259b3f7b0970/bazel/BUILD#L407-L430) file. 
```

Comment:
I'd suggest replacing first the "wasm" with "WebAssembly (Wasm)" to help with discovery when grepping, etc.

Also, we've switched from "runtime" to "engine" in Proxy-Wasm (https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/256), since we're using custom Proxy-Wasm runtime envioronment, and not any provided by those projects (which often bundle both: engine and runtime).

Reply:
Oh, and you should list available values here instead of linking to a specific commit, since it's going to get outdated.